Makefile.PL

```diff
@@ -8,7 +8,7 @@ license 'perl';
 perl_version '5.014002';
 
 requires
-  'Zonemaster'               => 0,
+  'Zonemaster::Engine'       => 0,
```

Comment:
Maybe up the version requirement to 2.0?

Reply:
Fixed.